### DISCUSSION TITLE
Table: Search option

### DISCUSSION THREAD

**Discussion by Tafsiralam** [grafana/grafana#69249] [Dashboards, Panels & Visualizations]
I am using **Athena Datasource** in Grafana to query S3 data using Athena directly on Grafana. But I am missing two main features:
**1.** Text Wrap - It's helpful in case, you have a long message.
**2.** Search among the results of a query.

I am not able to search among the results of a query. 

<img width="1981" alt="Screenshot 2023-05-30 at 4 22 48 PM" src="https://github.com/grafana/grafana/assets/36539677/552486c6-3f99-4fc8-bf56-9e370ab22269">

For **point-1**, I can see the feature request is added and, the team is already working on it. 
**URL:** https://github.com/grafana/grafana/issues/25623

For the **2nd point**, I am not seeing any feature request.
It would be great if we can add the search option above the table. This will help search among the results of a query.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82632. Feel free to continue the discussion around this there. Thank you!
